### ISSUE TITLE
Is it possible to disable server explorer?

### ISSUE THREAD

**Issue by emc211** [jshinonome/vscode-q#42]
Hey just started using this extension. Really helpful, thanks a lot for sharing.

Had one problem that seems to be takes very long time on initial connection to server to recursively build  the server explorer table, code for that appears to be defined here: https://github.com/jshinonome/vscode-q/blob/be600e1e33242822a9765facc4dd27b6f0ae2909/src/client/items/q-dict.ts#L105

Lazy fix would be if someway to have option to disable this server explorer.

Will take a closer look to see what part is taking so long and try and replicate.

Thanks

**Comment by emc211** [comment]
Appears to be caused by a large map of symbols maintained in the server i'm connecting to.
To replicate run this in server connecting to:
```
.ns.veryLargeDict:{x!x}(10000000?`4) 
```

Recursive line causing issue:
```
q)\ts r: r upsert (,/) {if[11h=type n:key x;r:(y[x] n), 'x];if[count r;r,:(,/).z.s[;y]each r[;0]];:r}[;f] each n;
16977 1260824864
```
Maybe a better idea than disabling would be to be stricter to ensure only namespaces that are traversed and not dictionaries.
```
r: r upsert (,/) {if[((::)~first x) and 11h=type n:key x;r:(y[x] n), 'x];if[count r;r,:(,/).z.s[;y]each r[;0]];:r}[;f] each n;
```

**Comment by jshinonome** [comment]
Thank you for providing details about the issue. Based on your investigation result, I have updated the query just released version 2.2.6. Let me know if it works.
